card/lib/card/migration.rb

```diff
@@ -42,8 +42,10 @@ def schema_mode mig_type=type
       original_suffix = ActiveRecord::Base.table_name_suffix
 
       ActiveRecord::Base.table_name_suffix = new_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
       yield
       ActiveRecord::Base.table_name_suffix = original_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
     end
```

Comment:
Rails 4 introduced an ActiveRecord model for the migration table. So changing the table name suffix is not sufficient to change the migration table. The SchemaMigration class remains connected to the old table.